packages/mobile-app/data/wallet/wallet.ts

```diff
@@ -999,6 +999,80 @@ export class Wallet {
     };
   }
 
+  async sendTransactionWithSpends(
+    network: Network,
+    accountName: string,
+    transactionVersion: number,
+    transactionFee: string,
+    outputs: Output[],
+    spendComponents: {
+      note: string;
+      witnessRootHash: string;
+      witnessTreeSize: string;
+      witnessAuthPath: {
+        side: "Left" | "Right";
+        hashOfSibling: string;
+      }[];
+    }[],
+  ) {
+    assertStarted(this.state);
+    let lastTime = performance.now();
+    // Make sure the account exists
+    const account = await this.state.db.getAccount(accountName);
+    if (account == null) {
```

Comment:
nit (non-blocking): I assume you want `===`, but no biggie. Could add a lint rule for that later.

Reply:
Actually this is intentional, I just habitually use this to as shorthand for `account === null || account === undefined`. But no strong preference here if we want to throw in a lint rule for it